lib/src/widgets/yaru_checkbox.dart

```diff
@@ -183,7 +182,71 @@ class _YaruCheckboxState extends YaruTogglableState<YaruCheckbox> {
 
   @override
   Widget build(BuildContext context) {
-    return buildToggleable(_YaruCheckboxPainter());
+    final checkboxTheme = YaruCheckboxTheme.of(context);
+    final painter = _YaruCheckboxPainter();
+    fillPainterDefaults(painter);
+
+    final unselectedState = <MaterialState>{};
+    final selectedState = unselectedState.union({MaterialState.selected});
+    final disabledState = unselectedState.union({MaterialState.disabled});
+    final selectedDisabledState = selectedState.union({MaterialState.disabled});
```

Comment:
the unions with an empty set are unnecessary. we could make these compile-time constants?

Reply:
Ofc makes sense :+1: